### PR TITLE
Refactor load_schedule.js

### PR DESCRIPTION
**Pull request by mickey8820062008**
I try to separate the code into proper functions and remove some duplicate code.
But the code may still be a little bit confused due to the naming of html contents looks so alike.

### CODE REVIEW

File: static/js/load_schedule.js [hunk 1]
```diff
@@ -1,75 +1,47 @@
-var last_schedule_id = "no_id"
-function load_schedule()
-{
+$(window).on('load',function(){
+    window.setInterval(load_schedule, 3000);
```
Comment (Billy4195) [inline static/js/load_schedule.js]: Why you change 1 sec to 3 sec??

Comment (mickey8820062008) [reply static/js/load_schedule.js]: fixed it... dumb mistake

File: static/js/load_schedule.js [hunk 2]
```diff
@@ -1,75 +1,49 @@
...
-                $('img#pic').css('display','none');
-                $('div.like_count').css('display','none');
-                $('img#event_qrcode').css('display','none');
-                $('div#user_pref').css('display','none');
-                $('div#news').css('display','none');
-                $('img#event_qrcode').css('display','none');
-                $('img#qrcode1').css('display','none');
-                $('img#qrcode2').css('display','none');
-                
                 console.log("text");
```
Comment (icecat2012) [inline static/js/load_schedule.js]: Is here missing resetTextDisplay()?

Comment (icecat2012) [reply static/js/load_schedule.js]: BTY, the first alphabet on commit message should be capitalized

File: static/js/load_schedule.js [hunk 3]
```diff
@@ -1,75 +1,50 @@
...
-                        $('div#name b').text(jsonRes.file_text.nickname);
-                        $('div#star b').text(jsonRes.file_text.constellation.name);
-                        $('div#overall b').text(jsonRes.file_text.constellation.value[0]);
-                        $('div#love b').text(jsonRes.file_text.constellation.value[1]);
-                        $('div#career b').text(jsonRes.file_text.constellation.value[2]);
-                        $('div#wealth b').text(jsonRes.file_text.constellation.value[3]);
+                        setDateInfo();
+                        setConstellationInfo(jsonRes);
                         if( jsonRes.file_text.news.length > 0){
                             console.log("print!!!");
```
Comment (icecat2012) [inline static/js/load_schedule.js]: Please help us, remove `console.log("print!!!");` in line 49

Comment (mickey8820062008) [reply static/js/load_schedule.js]: Actually, I wanna replace the code from line 49 to line 52 with function.
But I don't know how to name the function differ from the forum news below.

File: static/js/load_schedule.js [hunk 4]
```diff
@@ -1,180 +1,233 @@
...
+    }
+    else{
+        $('#location').html("")
+    }
+    if(jsonRes.file_text.detailtime != ""){
+        $('#detailtime').html("時間： " + jsonRes.file_text.detailtime)
+    }
+    else{
+        $('#detailtime').html("")
+    }
```
Comment (icecat2012) [inline static/js/load_schedule.js]: When this commit merge back to master. Will the conflict appear? Or the time & location bug will be back?

Comment (Billy4195) [reply static/js/load_schedule.js]: @stvreumi I think you will be more familiar with this part than me.

Comment (icecat2012) [reply static/js/load_schedule.js]: I try to switch the branch to see what will happen. Fortunately, the conflict will appear.
Although, the function part, which cause the bug reborn wont be conflict part. 
We can avoid the bug come back if we remember to modify the function part when solving the conflict.

Comment (Billy4195) [reply static/js/load_schedule.js]: @icecat2012 Did you mean that the bug won't appear again when dev branch merge into master branch?

Comment (icecat2012) [reply static/js/load_schedule.js]: The bug will appear again when merging back to master.
However, there will got a conflict when trying merge back to master.
So we can debug when solving the conflict.

Comment (Billy4195) [reply static/js/load_schedule.js]: Ok, got it.

Comment (mickey8820062008) [reply static/js/load_schedule.js]: So, should I modify the code to solve the existing bug or...?

Comment (mickey8820062008) [reply static/js/load_schedule.js]: I update the code from the closed PR.
I believe it should fix the bug.

**Pull request merged**